src/app/services/ccd-config/ccd-case.config.ts

```diff
@@ -96,10 +96,6 @@ export class AppConfig extends AbstractAppConfig {
     return this.config.print_service_url;
   }
 
-  public getRemotePrintServiceUrl() {
```

Comment:
I think this makes it a breaking change...? That would mean it needs to go up a major version.

Reply:
Agree with @paul-graham. Would this not break the build?

Reply:
@paul-graham I've now re-introduced this in `app.config.ts` in the CCD UI Toolkit as a deprecated, no-op function that returns `undefined`, ahead of its removal in the next major release.